scheme/psa-iot/extractor.go

```diff
@@ -113,7 +117,7 @@ func extractMeasurement(obj MeasurementExtractor, m comid.Measurement, class Cla
 		return &proto.Endorsement{}, fmt.Errorf("failed to create software component attributes: %w", err)
 	}
 	refVal := proto.Endorsement{
-		Scheme:     schemeName,
+		Scheme:     SchemeName,
```

Comment:
As discussed: this needs working through to resolve the correct schemeName as this part of the code is used by both PSA and CCA